css/style.css

```diff
@@ -15,5 +26,177 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+/* header */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2rem 5rem;
+}
+.logo {
+  display: block;
+  min-width: 2rem;
```

Comment:
If you give the `img` inside it, a width and height in the HTML, you won't need `min-width` and `min-height`